jsettlers.tools/build.gradle

```diff
@@ -23,4 +24,12 @@ jar {
     }
 }
 
-tasks.run.workingDir = rootProject.projectDir
\ No newline at end of file
+tasks.run.workingDir = rootProject.projectDir
+
+project.extensions.findByName('checkstyle').sourceSets.each { SourceSet s ->
```

Comment:
from what I can see the checkstyle file belongs to config/checkstyle/checkstyle.xml. Then it is no longer necessary to do the configuration stuff.
Also the plugin should probably be added to all submodules and not just jsettlers.tools

Reply:
I totally agree. But as I am not a gradle guru I was happy having added it to one project. Could you help move that stuff on a global level?

Reply:
Neither am I. I just did a quick google search and found [this stackoverflow question](https://stackoverflow.com/questions/26236308/how-to-apply-plugin-to-allprojects-with-new-gradle-plugins-mechanism)